Python/ast.c

```diff
@@ -3527,13 +3608,23 @@ ast_for_suite(struct compiling *c, const node *n)
     return seq;
 }
 
+static void
+get_last_end_pos(asdl_seq *s, int *end_lineno, int *end_col_offset)
+{
+    int tot = asdl_seq_LEN(s);
+    stmt_ty last = asdl_seq_GET(s, tot - 1);
```

Comment:
Does this need a check that `tot > 0`?

Reply:
Added this just in case (to be future proof against any AST optimizers).